src/gui/AboutDialog.cpp

```diff
@@ -209,91 +213,40 @@ static const QString aboutContributors = R"(
-        lucasjsoliveira, mauri.andres, newmanisaac, rafaelnp, ruanmed, rudahximenes, ul1sses, vitor895, weslly, wtuemura, xendez,
-        zodSilence</li>
-    <li><strong>Português (Portuguese (Portugal))</strong>: a.santos, American_Jesus, arainho, hds, hugok, lecalam, lmagomes, pfialho,
-        smarquespt, smiguel, xendez, xnenjm</li>
-    <li><strong>Română (Romanian)</strong>: _parasite_, aduzsardi, alexminza, polearnik</li>
-    <li><strong>русский (Russian)</strong>: 3nad, _nomoretears_, agag11507, alexandersokol, alexminza, anm, artemkonenko, ashed,
-        BANOnotIT, burningalchemist, cl0ne, cnide, denoos, DG, DmitriyMaksimov, egorrabota, injseon, Japet, JayDi85, KekcuHa, kerastinell,
-        laborxcom, leo9uinuo98, Mogost, Mr.GreyWolf, MustangDSG, netforhack, NetWormKido, nibir, Olesya_Gerasimenko, onix, Orianti,
-        RKuchma, ruslan.denisenko, ShareDVI, Shevchuk, solodyagin, talvind, treylav, upupa, VictorR2007, vsvyatski, wakeeshi, Walter.S,
-        wkill95, wtigga, zOrg1331</li>
-    <li><strong>српски језик (Serbian)</strong>: ArtBIT, ozzii</li>
-    <li><strong>Slovenčina (Slovak)</strong>: Asprotes, crazko, jose1711, l.martinicky, pecer, reisuya, Slavko</li>
-    <li><strong>Slovenščina (Slovenian)</strong>: asasdasd, samodekleva</li>
-    <li><strong>Español (Spanish)</strong>: adolfogc, antifaz, capitantrueno, cquike, cyphra, DarkHolme, doubleshuffle, e2jk,
-        EdwardNavarro, fserrador, gabeweb, gonrial, jjtp, jorpilo, LeoBeltran, mauri.andres, piegope, pquin, puchrojo, rodolfo.guagnini,
-        tierracomun, vsvyatski</li>
-    <li><strong>Svenska (Swedish)</strong>: 0x9fff00, aiix, Anders_Bergqvist, ArmanB, Autom, baxtex, eson, henziger, jpyllman, malkus,
-        merikan, peron, peterkz, Thelin, theschitz, victorhggqvst</li>
-    <li><strong>ไทย (Thai)</strong>: arthit, ben_cm, chumaporn.t, darika, digitalthailandproject, GitJirasamatakij, ll3an, minoplhy,
-        muhammadmumean, nimid, nipattra, ordinaryjane, rayg, sirawat, Socialister, Wipanee</li>
-    <li><strong>Türkçe (Turkish)</strong>: abcmen, ahmed.ulusoy, cagries, denizoglu, desc4rtes, etc, ethem578, kayazeren, mcveri, N3pp,
-        rgucluer, SeLeNLeR, sprlptr48, TeknoMobil, Ven_Zallow, veysiertekin</li>
-    <li><strong>Українська (Ukrainian)</strong>: brisk022, chulivska, cl0ne, exlevan, m0stik, moudrick, netforhack, olko, onix, paul_sm,
-        ShareDVI, upupa, zoresvit</li>
+    <li><strong>Arabic:</strong> kmutahar</li>
+    <li><strong>Chinese (China):</strong> Brandon_c, ligyxy, Biggulu, hoilc, Small_Ku, vc5, umi_neko</li>
+    <li><strong>Chinese (Taiwan):</strong> ymhuang0808, BestSteve, flachesis, MiauLightouch, yan12125, Small_Ku</li>
+    <li><strong>Czech:</strong> tpavelek, DanielMilde, pavelb</li>
+    <li><strong>English (United Kingdom):</strong> YCMHARHZ</li>
+    <li><strong>English (United States):</strong> alexandercrice, DarkHolme, nguyenlekhtn</li>
+    <li><strong>Finnish:</strong> MawKKe, artnay</li>
```

Comment:
I'm removed from this list?

Reply:
Umm. I did not do that on purpose. I only took what the API gave me. I did blacklist @droidmonkey and myself, because we're like coordinators for every language. Perhaps you're not enrolled as a translator anymore?

Reply:
That's strange. I do often translations in Transifex, although it has been a while since I did any.

Reply:
Got it. I'm on the reviewers list instead of translators.

Reply:
Yes they are equally important